CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Smart Order Routers
```

Comment:
SORs -> MM

Reply:
Addressed in: fe42fd709a4553436c6dc517fd7dd24f8f002591